site/src/pages/RoadmapPage/RoadmapMultiplan.tsx

```diff
@@ -89,18 +94,12 @@ const RoadmapMultiplan: FC = () => {
     setEditIdx(-1);
   };
 
-  const duplicateCurrentPlan = () => {
-    const currentPlan = allPlans.plans[currentPlanIndex];
-    let newName = `${currentPlan.name} (Copy)`;
-    let counter = 1;
-    while (allPlans.plans.find((p) => p.name === newName)) {
-      counter++;
-      newName = `${currentPlan.name} (Copy ${counter})`;
-    }
```

Comment:
Copying does not seem to increment the numbers correctly after this change. I assume it's because of this.
<img width="472" alt="image" src="https://github.com/user-attachments/assets/2b4b5080-1733-43a0-81bf-031046460214" />